src/views/ProfilePage/Profile.js

```diff
@@ -0,0 +1,338 @@
+                type="text"
+                handleChange={this.handleChange}
+                label="Username"
+                error={errors.username}
+              />
+
+              <TextArea
+                name="bio"
+                value={bio || ''}
+                type="text"
+                label="Bio"
+                error={errors.email}
+                handleChange={this.handleChange}
+                style={{
+                  height: '80px',
+                  width: '300px',
+                }}
+              />
+              <Button
+                label={loading ? null : 'Save'}
+                handleClick={this.handleSubmit}
+                disabled={loading ? true : !isFormValid}
+                type="submit"
+                datatest="submit-button"
+                style={{
+                  height: '45px',
+                  width: '500px',
+                  marginLeft: '25px',
+                  color: '#ffc700',
+                  backgroundColor: '#000',
+                  borderRadius: '0',
+                }}
+              >
+                {loading && loader}
+              </Button>
+              <img src={image} alt="" className="form-image" />
+              <input type="file" id="image" name="file" className="profile-img" onChange={this.handleImageChange} />
+            </form>
+          </Dialog>
+        ) : (null)}
+
+        <div className="profile-container">
+          <div className="sidebar">
+            <ul>
+              <li><a href="/">New Article</a></li>
+              <li><a href="/">Bookmarks</a></li>
+              <li><a href="/">Account</a></li>
+              <li><a href="/">Stats</a></li>
+              <li><a href="/">Notification</a></li>
+              <li><a href="/">Help</a></li>
+              <li><a href="/">Log Out</a></li>
+            </ul>
+          </div>
+          <div className="main">
+            <div className="profile-header">
+              <div className="profile-details">
+                <div className="name-button">
+                  <h3>{`${firstname} ${lastname}`}</h3>
+                  <span><Button handleClick={this.handleToggleEditProfileModal} datatest="edit-button">Edit Profile</Button></span>
+                  <span><p>{`@${username}`}</p></span>
+                </div>
+                <p>
+                  { bio }
+                </p>
+                <p className="following-follower">
+                  <span className="following">{`${followingCount} Following`}</span>
+                  <span className="followers">{`${followerCount} Followers`}</span>
+                </p>
+              </div>
+              <div className="profile-image">
+                <img src={image} alt="" className="profile-image" />
+              </div>
+            </div>
+          </div>
+          <div />
+          <div className="profile-links">
+            <ul>
+              <li className="active">
+                <a href="/">Profile</a>
```

Comment:
Why isn't the React `Link` used?

Reply:
It was an oversight, I'll change it now.